### DISCUSSION TITLE
Alpha blend texture is not visible in iOS Quick-Look (auto generating USDZ)

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#2407] [General]
Hello friends,

After update of 1.7 we tested "auto generating USDZ" feature. 

Before 1.7 we were using USDZ converter via macOS terminal. It was working fine for alpha blend textures (glass, plastic etc)

With auto generating USDZ feature alpha blend texture is not converting. It converts alpha blend texture to normal texture. (without opacity)

Is there a kind of issue about that ? anyone tested ?

Thanks...

**Comment by elalish** [comment]
Yes, many features are not yet enabled. If auto-generation is not working, you'll have to fall back to `ios-src` as usual. Contributions to the three.js USDZExporter will solve this, so feel free to lead the charge!

**Reply by base-krchome** [reply]
Feedback : Actually you and your team managed to convert glb to usdz. This is awesome because of that your system converts DRAKO compression glb to usdz immedeatly. THIS IS GAME CHANGING MOVEMENT.... million polygons fit in 5 mb glb and you converting to usdz without any extra waiting(i observed that maybe i am wrong about that waiting time).  Well-done ladies and 
gentlemen. 

But our researches show that glass or alpha textures are in %15 percent of 3d model industry(maybe more but not less).. Glass, alpha logos on 3d models etc... I hope you guys can managed to success about it . 10 points goes to MODEL VIEWER!!!

**Reply by elalish** [reply]
@grorg Food for thought regarding format interop.

**Comment by canberka** [comment]
This converting feature is the most important bridge of model viewer... We are waiting for a full opening. We are followers. 

Thank you all of you.
